.github/workflows/ci.yml

```diff
@@ -306,6 +306,9 @@ jobs:
       - name: Install PHPStan
         run: composer bin phpstan install
 
+      - name: Install PHPBench
+        run: composer bin phpbench install
```

Comment:
same than https://github.com/zenstruck/foundry/pull/866/files#r2036127527 but for CI